plugins/language_json.lua

```diff
@@ -0,0 +1,27 @@
+-- mod-version:3 priority:110
+
+local syntax = require "core.syntax"
+
+syntax.add {
+  name = "JSON",
+  files = { "%.json$", "%.cjson$" },
+  comment = "//",
+  block_comment = {"/*", "*/"},
+  patterns = {
+  
+    -- cjson support
+    { pattern = "//.*", type = "comment" },
+    { pattern = { "/%*", "%*/" }, type = "comment" },
+    
+    { pattern = '"[^\n]*"%s*:', type = "keyword" }, -- key
+    { pattern = '"[^\n]*"', type = "string" }, -- value
```

Comment:
```suggestion
    { pattern = '"[^\n]-"%s*:', type = "keyword" }, -- key
    { pattern = '"[^\n]-"', type = "string" }, -- value
```

Reply:
Also, to avoid highlighting the `:`:
```suggestion
    { pattern = '"[^\n]*"%s*():', type = { "keyword", "normal"} }, -- key
    { pattern = '"[^\n]*"', type = "string" }, -- value
```